### DISCUSSION TITLE
Challenge question

### DISCUSSION THREAD

**Discussion by SamayraHN** [devchallenges-io/curriculum#1] [General]
Hello team,

When participating in a challenge, do we need to recreate the page shown i.e. be a carbon copy more or less or are we creating something similar to the page?

**Comment by nghiemthu** [comment]
Hi,

When participating in a design challenge, it is highly recommended to closely replicate the provided design, especially for the first page of your submission. This is important because your demo will be evaluated based on its similarity to the original design, and scoring will reflect how well you adhere to the guidelines.
